src/components/Image/index.native.tsx

```diff
@@ -39,25 +35,27 @@ function Image(props) {
             {...rest}
             source={imageSource}
             onLoad={(evt) => {
+                if (typeof source === 'number' || !source.uri) {
+                    return;
+                }
                 const {width, height} = evt.nativeEvent;
                 dimensionsCache.set(source.uri, {width, height});
-                if (props.onLoad) {
-                    props.onLoad(evt);
+                if (rest.onLoad) {
+                    rest.onLoad(evt);
                 }
             }}
         />
     );
 }
 
-Image.propTypes = imagePropTypes;
-Image.defaultProps = defaultProps;
 Image.displayName = 'Image';
-const ImageWithOnyx = withOnyx({
+Image.resizeMode = RESIZE_MODES;
+Image.resolveDimensions = resolveDimensions;
```

Comment:
`resizeMode` and `resolveDimensions` was set to `ImageWithOnyx`, not `Image`. What's the reason of this change?

Reply:
TypeScript says the following: `Property 'resizeMode' does not exist on type 'ComponentType<Omit<ImageProps, "session">>'.` when I try to apply these to **ImageWithOnyx** 😅

Reply:
I believe we can also remove `resolveDimensions` altogether, don't seems to be used anywhere.

Reply:
Done!